modules/custom/az_core/src/Plugin/migrate/process/ManualMigrationLookup.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\az_core\Plugin\migrate\process;
+
+use Drupal\Core\Database\Database;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\MigrateSkipProcessException;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Process Plugin to handle content that was manually migrated.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "az_manual_migration_lookup"
+ * )
+ * Only works when the source database is a Drupal 7 database.
+ *
+ * This plugin looks up content from the source database and returns the field
+ * data for the found entity. This requires an entity id from the source site.
+ * This is useful for content that was manually migrated (Copy and Pasted) and not
+ * tied to a migration.
+ *
+ * @return string source entity identifier Example: node title.
```

Comment:
Nit-picky suggestion:
```suggestion
 * @return string source entity label (Example: node title).
```